sharding-distribution/sharding-jdbc-distribution/pom.xml

```diff
@@ -129,17 +129,27 @@
         </dependency>
         <dependency>
             <groupId>org.apache.shardingsphere</groupId>
-            <artifactId>sharding-orchestration-core</artifactId>
+            <artifactId>sharding-orchestration-core-new</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.apache.shardingsphere</groupId>
-            <artifactId>sharding-orchestration-reg-api</artifactId>
+            <artifactId>sharding-orchestration-center-api</artifactId>
```

Comment:
Why not use
<artifactId>sharding-orchestration-center</artifactId>
It includes :
    <modules>
        <module>sharding-orchestration-center-api</module>
        <module>sharding-orchestration-center-apollo</module>
        <module>sharding-orchestration-center-zookeeper-curator</module>
        <module>sharding-orchestration-center-nacos</module>
    </modules>